data/types/food/food.go

```diff
@@ -9,15 +9,16 @@ import (
 const (
 	Type = "food"
 
-	BrandLengthMaximum     = 100
-	CodeLengthMaximum      = 100
-	MealBreakfast          = "breakfast"
-	MealDinner             = "dinner"
-	MealLunch              = "lunch"
-	MealOther              = "other"
-	MealOtherLengthMaximum = 100
-	MealSnack              = "snack"
-	NameLengthMaximum      = 100
+	BrandLengthMaximum        = 100
+	CodeLengthMaximum         = 100
+	MealBreakfast             = "breakfast"
+	MealDinner                = "dinner"
+	MealLunch                 = "lunch"
+	MealOther                 = "other"
+	MealOtherLengthMaximum    = 100
+	MealSnack                 = "snack"
+	MealHypoglycemiaTreatment = "hypoglycemiaTreatment"
```

Comment:
Alphabetical order, please. This needs to be between `MealDinner` and `MealLunch`.